src/main/java/com/suse/saltstack/netapi/datatypes/Job.java

```diff
@@ -1,17 +1,68 @@
 package com.suse.saltstack.netapi.datatypes;
 
+import com.google.gson.annotations.SerializedName;
 import java.util.List;
 
+// todo - handle dates too (they are represented in really
```

Comment:
Minor detail but could you use the `TODO:` syntax for todos. Its more common and gets recognized by most IDEs and tools.